Sources/SKSwiftPMWorkspace/SwiftPMWorkspace.swift

```diff
@@ -221,13 +228,13 @@ extension SwiftPMWorkspace {
     /// with only some properties modified.
     self.packageGraph = packageGraph
 
-    self.fileToTarget = [AbsolutePath: TargetBuildDescription](
+    self.fileToTarget = [AbsolutePath: SwiftBuildTarget](
       packageGraph.allTargets.flatMap { target in
         return target.sources.paths.compactMap {
-          guard let td = plan.targetMap[target] else {
+          guard let buildTarget = BuildDescription(buildPlan: plan).getBuildTarget(for: target) else {
```

Comment:
Seems like the build description is used over the plan now, maybe just wrap it above rather than in here (and below)?

Reply:
Good idea 👍🏽